extensions/wikia/UserRenameTool/RenameUserProcess.class.php

```diff
@@ -610,21 +609,12 @@ private static function testBlock( $text ) {
 		$phalanxMatchParams = PhalanxMatchParams::withGlobalDefaults()->content( $text );
 		$phalanxService = PhalanxServiceFactory::getServiceInstance();
 
-		$blockFound = false;
-
-		foreach ( Phalanx::getSupportedTypeNames() as $blockType ) {
-			$phalanxMatchParams->type( $blockType );
-			$res = $phalanxService->doMatch( $phalanxMatchParams );
-
-			if ( !empty( $res ) ) {
-				$blockFound = true;
-				break;
-			}
-
-		}
+		$phalanxMatchParams->type( "user" );
```

Comment:
Minor coding conventions tweak:
```suggestion
		$phalanxMatchParams->type( 'user' );
```